test/e2e/upgrade/service/service.go

```diff
@@ -96,7 +96,10 @@ func (t *UpgradeTest) Setup(f *framework.Framework) {
 
 	// Hit it once before considering ourselves ready
 	ginkgo.By("hitting pods through the service's LoadBalancer")
-	timeout := service.LoadBalancerLagTimeoutAWS
+	timeout := service.LoadBalancerLagTimeoutDefault
```

Comment:
i'd rather just put 10m right here.  there is no scenario where i care about what upstream's value is.

Reply:
> i'd rather just put 10m right here...

Done with 0fd8f40010 -> 65bd1de636.